sirepo/package_data/template/srw/rsoptExport.sh.jinja

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
-echo "NOTE: remove files from {{ runDir }} first"
+if [ -d "{{ runDir }}" ] && [ -n "$(ls -A {{ runDir }})" ]; then
+   echo "Cleaning run directory {{ runDir }}..."
+   rm {{ runDir }}/{{ pyFileName }}
+   rm {{ runDir }}/run_parallel_python.py
+fi
+echo "Generating data..."
 SECONDS=0
 rsopt sample configuration "{{ ymlFileName }}" > "{{ outFileName }}" 2>&1
 if [ $? -eq 0 ]
 then
-    echo "Completed scan in $SECONDS sec"
-    echo "NOTE: python {{ pyFileName }} rsopt_run" `tail -1 "{{ outFileName }}" | cut -d: -f2`.npy "to run SRW with generated data"
+    echo "Completed generation in $SECONDS sec"
+    run_py="python {{ pyFileName }} rsopt_run `tail -1 {{ outFileName }} | cut -d: -f2`.npy"
+    read -p "Continue? (** will run SRW {{ totalSamples }} times **)? (Y/N): " confirm
```

Comment:
Why do we want to prompt the user here? They could just control-C. If they are running this with `&`, they will have to answer a prompt. Was this a user request?

Reply:
Not a user request, just thought an explicit opt-out would be polite. This also allows printing the instructions on "N". Not married to it

Reply:
Or, for another hot take, if they know to use ```&``` then they can use ```yes ```  :)